clang/lib/Sema/SemaType.cpp

```diff
@@ -8803,6 +8821,12 @@ static void processTypeAttrs(TypeProcessingState &state, QualType &type,
       attr.setUsedAsTypeAttr();
       break;
     }
+    case ParsedAttr::AT_HLSLResourceClass:
```

Comment:
Could you add a TODO to add more cases for the attributes that will be added here:
https://github.com/llvm/wg-hlsl/issues/39
I think there will be others too, right? Like is_ms / is_feedback potentially?

Reply:
Correct, we will be adding quite a few new attributes here. I would prefer not to add the TODOs though. There are many other places that need to be modified when a new attribute is added, and I don't want to add TODO everywhere. I think it is better to just look for where the existing attributes are processed when adding a new one.